internal/services/healthcare/healthcare_dicom_resource.go

```diff
@@ -115,6 +118,86 @@ func resourceHealthcareApisDicomService() *pluginsdk.Resource {
 				Default:  true,
 			},
 
+			"cors_configuration": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"allowed_origins": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type:         pluginsdk.TypeString,
+								ValidateFunc: validation.StringIsNotEmpty,
+							},
+						},
+						"allowed_headers": {
```

Comment:
```suggestion
						},

						"allowed_headers": {
```
please add blank lines between properties

Reply:
Thanks, updated.